README.md

```diff
@@ -6,6 +6,28 @@
 
 Feature selection for scikit-learn estimators
 
+### Overview:
+If you have encountered a database with so many features, that while working on it and gets messy, a good idea is to approach this problem by selecting only some of these features for your model. Feature seletion will reduce complexity, reduce the time when training an algorithm, and improve the accuracy of your model (if we select them wisely). However, this is not a trivial question. 
+
+To help you out performing this task, we have created the **feature-selection** package in `python`.
+
+If you are interested in a similar feature selection package for `R`, click [here](https://github.com/UBC-MDS/feature-selection-r).
+
+### Feature description:
+In this package, four functions are included to lead you with feature selection:
+
+#### Forward
+- Interative algorithm that starts as an empty model, add features that improves the accuracy of the model, and stops when the accuracy doesn't improve anymore.  
```

Comment:
Could use a bit more details? Lots of examples online, such as this [one](https://www.sciencedirect.com/topics/computer-science/forward-selection), to get some ideas from. Doesn't need to be much more comprehensive.

Reply:
Ok, I just did it. Thanks for the link to the examples.